Library/Homebrew/test/support/helper/test_case.rb

```diff
@@ -16,12 +16,17 @@ class TestCase < ::Minitest::Test
 
     def setup
       super
+
       @__argv = ARGV.dup
+      @__env = copy_env # Call #to_hash to duplicate ENV
```

Comment:
Oh, that's a leftover from when I inlined the `ENV.to_hash`. (I actually put it in a named function to avoid the need for the comment!) So I think I'll just remove the comment.